sound/soc/intel/boards/sof_sdw.c

```diff
@@ -409,6 +411,7 @@ static struct sof_sdw_codec_info codec_info_list[] = {
 		.dai_name = "rt711-aif1",
 		.init = sof_sdw_rt711_init,
 		.exit = sof_sdw_rt711_exit,
+		.codec_type = SOF_SDW_CODEC_TYPE_JACK,
```

Comment:
in the case of RT711, we may want to be careful: it also has local microphone support, and more generally when we move to SDCA parts we can have devices with multiple Functions. So I wonder if we should treat the 'codec_type' as 'function_mask' to allow a device to have multiple functions.

We can also revisit this later since we would also need to have multiple dai names so there's still work to be done here.